DICOMPlugins/DICOMParametricMapPlugin.py

```diff
@@ -71,7 +71,7 @@ def load(self,loadable):
     logging.debug('DICOM PM load()')
     try:
       uid = loadable.uid
-      logging.debug('in load(): uid = ', uid)
+      logging.debug('in load(): uid = %s', uid)
```

Comment:
This isn't the correct syntax; would need to use % to format the string.  Previously would just print the value, which was the expected behavior.  Modern python would prefer to use the format string `f"uid = {uid}"`

Reply:
The original version `logging.debug('in load(): uid = ', uid)` was not valid syntax.

The updated `logging.debug('in load(): uid = %s', uid)` is correct and fast.

Using f-string is correct, more readable, and slow, as string formatting and all expressions that are used in the string are always evaluated at runtime, regardless the log message is filtered out or not. See for example here https://blog.pilosus.org/posts/2020/01/24/python-f-strings-in-logging/).

In this case the performance does not matter, but for consistency and to avoid the risk of slow code getting into performance-critical places by copy-pasting, I would not use f-string for log message formatting.

Reply:
Yes, I see, it's `logging` not `print` so the arguments are different.  The correction is fine then 👍